ledger-core/virtual/integration/delegationtoken_test.go

```diff
@@ -28,6 +28,89 @@ import (
 	"github.com/insolar/assured-ledger/ledger-core/virtual/testutils"
 )
 
+var messagesWithToken = []struct {
```

Comment:
it means, they may have tokens?

Reply:
Yes, this is list  of messages which can have token